$NetBSD: patch-ca,v 1.4.6.1 2012/04/18 13:50:48 tron Exp $

* Before calling sigsend(), make sure the function actuall exists.
  Fixes the build on NetBSD >= 4.99.49, in which the macro P_MYID
  is defined somewhere unrelated to signals.

--- configure.in.orig	2011-01-25 11:27:51.000000000 +0000
+++ configure.in
@@ -2051,7 +2060,7 @@ AC_CHECK_HEADERS(varargs.h stdarg.h dire
 paths.h semaphore.h)
 
 # Already-done: strcasecmp
-AC_CHECK_FUNCS(lstat putenv select setenv setlocale strcoll tcgetattr)
+AC_CHECK_FUNCS(lstat putenv select setenv setlocale sigsend strcoll tcgetattr)
 
 AC_STAT_MACROS_BROKEN
 MYSQL_SIGNAL_CHECK
